cordova-lib/src/cordova/plugin.js

```diff
@@ -154,6 +135,38 @@ module.exports = function plugin(command, targets, opts) {
                             is_top_level: true
                         };
 
+                        // If no version is specified, retrieve the version (or source) from config.xml
+                        if (!version && !cordova_util.isUrl(id) && !cordova_util.isDirectory(id)) {
+                            events.emit('verbose', 'no version specified, retrieving version from config.xml');
+                            var ver = getVersionFromConfigFile(id, cfg);
```

Comment:
I know this is just moved code, but while you're here could you make that sentence start with an uppercase "N" (as in "No version specified...")? :smile: